subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -1,179 +1,194 @@
 # SPDX-License-Identifier: Apache-2.0
 
+if(CONFIG_BT_CTLR_ADVANCED_FEATURES)
+  message(WARNING "\nCONFIG_BT_CTLR_ADVANCED_FEATURES=y, Advanced Features' "
+                  "default value change could change Zephyr Bluetooth "
+                  "Controller's functional behavior.")
+endif()
+
 zephyr_library()
+
 zephyr_library_sources(
   util/mem.c
   util/memq.c
   util/mayfly.c
   util/dbuf.c
   util/util.c
   ticker/ticker.c
-  ll_sw/ll_addr.c
+  ll_sw/ll_feat.c
   ll_sw/ll_tx_pwr.c
+  ll_sw/ll_addr.c
+  ll_sw/ull.c
+  ll_sw/lll_common.c
   hci/hci_driver.c
   hci/hci.c
   )
 
-zephyr_library_sources_ifdef(
-  CONFIG_BT_CTLR_CRYPTO
-  crypto/crypto.c
-  )
+if(CONFIG_BT_BROADCASTER)
+  zephyr_library_sources(
+    ll_sw/ull_adv.c
+    )
+  zephyr_library_sources_ifdef(
+    CONFIG_BT_CTLR_ADV_EXT
+    ll_sw/ull_adv_aux.c
+    )
+  zephyr_library_sources_ifdef(
+    CONFIG_BT_CTLR_ADV_PERIODIC
+    ll_sw/ull_adv_sync.c
+    )
+  zephyr_library_sources_ifdef(
+    CONFIG_BT_CTLR_ADV_ISO
+    ll_sw/ull_adv_iso.c
+    )
+endif()
 
-if(CONFIG_BT_LL_SW_SPLIT)
```

Comment:
i should not remove this!

Reply:
from subsys/bluetooth/CMakeLists.txt:

```
if(CONFIG_BT_CTLR AND CONFIG_BT_LL_SW_SPLIT)
  add_subdirectory(controller)
endif()
```

so yes, removing this should be ok